blocks/math.js

```diff
@@ -32,32 +32,24 @@ goog.require('Blockly.Colours');
 
 goog.require('Blockly.constants');
 
-
-/**
- * Common HSV hue for all blocks in this category.
- */
-Blockly.Blocks.math.HUE = Blockly.Colours.textField;
-
 Blockly.Blocks['math_number'] = {
   /**
    * Block for generic numeric value.
    * @this Blockly.Block
    */
   init: function() {
```

Comment:
(Doesn't have to be done in this pr)

Your math blocks are different from the default math blocks in enough ways--mostly involving tooltips and helpurls--that you should move them into blocks_horizontal/ or blocks_vertical.  Otherwise we may be fighting merge conflicts over and over again.